src/main/java/com/iotiq/application/service/OrderService.java

```diff
@@ -42,13 +44,14 @@ public List<Order> getOrdersBySeller() {
     }
 
     public OrderCreateResponse createOrder(OrderCreateRequest createRequest) {
-        Order order =  ModelMapperUtil.map(createRequest, Order.class);
+        Order order = ModelMapperUtil.map(createRequest, Order.class);
 
         Order finalOrder = order;
         order.setOrderedProducts(createRequest.getCartItems()
                 .stream().map(cartItem -> convertToOrderedProduct(cartItem, finalOrder)).collect(Collectors.toList()));
         order.setCustomer(customerService.getCurrentCustomer());
         order.setTotalPrice(calculateTotalAmount(order.getOrderedProducts()));
+        order.setOrderDate(LocalDateTime.now(ZoneOffset.UTC));
```

Comment:
We need relative time firstly for displaying, filtering without in-memory operations.

So we should remove "ZoneOffset.UTC":

```suggestion
        order.setOrderDate(LocalDateTime.now());
```

 if we want to store utc time, storing in new db field is better.

![image](https://github.com/user-attachments/assets/e5e8148e-f4dc-4d3f-beda-9553985b18fc)